### PR TITLE
Basic integration tests for amp-story

### PR DESCRIPTION
**Pull request by alanorozco**

### CODE REVIEW

File: test/integration/test-amp-story.js [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+          .then(() => expectActivePageAfterWait('page-1', TIME_PAGE_0))
+          .then(() => expectActivePageAfterClickForward('page-2'))
+          .then(() => {
+            clickForward();
+            return expectActiveBookend();
+          }))
+        .timeout(TIME_ADVANCE_TIMEOUT);
+  });
+
+  // TODO(alanorozco): Test for media-based advancement
```
Comment (aghassemi) [inline test/integration/test-amp-story.js]: Heads up that SauceLabs uses VM's that can't play most videos that require hardware acceleration. (https://support.saucelabs.com/hc/en-us/articles/115002202568-Unsupported-Video-Types). Might be a good time to consider using real devices ( https://github.com/ampproject/amphtml/issues/7822 ) @rsimha-amp

Comment (rsimha-amp) [reply test/integration/test-amp-story.js]: Noted. Will try to look at this soon.

**Comment by newmuis** [comment]
@alanorozco just curious: what were the hurdles to submitting this?  If it needs work, I may be able to pick up where you left off.

**Comment by alanorozco** [comment]
@newmuis I got myself lost in a bunch of notifications :(

I got blocked on some Saucelabs tests that were failing, and I've been dropping this as a result of other priorities.

Feel free to take on. I might have some time this week to take a look at this, so if you do end up taking it, or have already written integration tests yourself, please let me know.

**Comment by aghassemi** [comment]
Thanks for the pull request. Since this PR has been stale for a while, I am closing it. Please feel free to reopen as needed.

**Pull request closed without merging**